src/3d/qgslayoutitem3dmap.cpp

```diff
@@ -26,9 +26,12 @@
 QgsLayoutItem3DMap::QgsLayoutItem3DMap( QgsLayout *layout )
```

Comment:
It'd be nice if this layout item had a dedicated icon to be used in the items panel, i.e. here:
![image](https://user-images.githubusercontent.com/1728657/44616700-a73df000-a87e-11e8-8d6d-4bd38241d44e.png)

```
QIcon QgsLayoutItem3DMap::icon() const
{
  return QgsApplication::getThemeIcon( QStringLiteral( "/_______________.svg" ) );
}
```